ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -41,15 +41,16 @@
             label: `.${Tabs.bemBlock}__label`,
             description: `.${Tabs.bemBlock}__longdescription`,
             qm: `.${Tabs.bemBlock}__questionmark`,
-            tooltipDiv: `.${Tabs.bemBlock}__shortdescription`
+            tooltipDiv: `.${Tabs.bemBlock}__shortdescription`,
+            olTabList: `.${Tabs.bemBlock}__tablist`
         };
 
         constructor(params) {
             super(params);
             const {element} = params;
+            this._templateHTML = {};
             this.#cacheElements(element);
-            this.#cacheTabPanelElementIds();
-            this._active = this.getActiveIndex(this._elements["tab"]);
+            this._active = this.getActiveTabId(this.#getCachedTabs());
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
Ok, thanks , not sure why still its failing by saying define the variable inside a class. Checking this ,will resolve